.github/workflows/rust.yml

```diff
@@ -12,6 +12,24 @@ defaults:
     working-directory: ./rust
 
 jobs:
+  static-analysis:
+    runs-on:
+      - ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Update toolchain
+        run: rustup show
+      - uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: ./rust
+          save-if: ${{ github.ref == 'refs/heads/main' }}
+      - run: cargo fmt -- --check
+      - run: |
+          cargo doc --all-features --no-deps --document-private-items ${{ matrix.packages }}
```

Comment:
There is no `matrix` in this job.

Reply:
Should have waited for this to be reviewed further. See https://github.com/firezone/firezone/pull/2298